asset_dashboard/templates/asset_dashboard/projects.html

```diff
@@ -34,4 +34,8 @@ <h3>Capital Projects</h3>
       </tbody>
     </table>
   </div>
+<<<<<<< HEAD
 {% endblock %}
+=======
+{% endblock %}
+>>>>>>> Created a project list view, detail view, and form.
```

Comment:
I don't know where this came from. It's not on my local version. I can't seem to pull this down so that I can delete this. Trying to figure out what's up...

Reply:
If you view the file in github, then you can see it: https://github.com/datamade/ccfp-asset-dashboard/blob/75eb849f958dd4663c664f2f629bfaefea081f01/asset_dashboard/templates/asset_dashboard/projects.html

But that text is not there if you try and "edit" the file: https://github.com/datamade/ccfp-asset-dashboard/edit/project-form/asset_dashboard/templates/asset_dashboard/projects.html?pr=%2Fdatamade%2Fccfp-asset-dashboard%2Fpull%2F17

Reply:
And it's not in the diff for the PR. https://github.com/datamade/ccfp-asset-dashboard/pull/17/files